types/Checkbox.php

```diff
@@ -62,8 +67,11 @@ public function multiValueEditor($name, $rawvalues) {
                 $checked = '';
             }
 
+            $id = !empty($htmlID) ? "id=\"$htmlID\"" : '';
+            $htmlID = null;
```

Comment:
This is a loop. I only want to give the ID to the first checkbox, all the further checkboxes should not receive an ID. Hence I assign `$htmlID` `null` so that the check `!empty($htmlID)` in the line above fails.